### PR TITLE
Use external non-shared network for gateway

### PR DESCRIPTION
**Pull request by almahmoud**
Addresses https://github.com/CloudVE/cloudbridge/issues/306

### CODE REVIEW

File: cloudbridge/providers/openstack/services.py [hunk 1]
```diff
@@ -1249,7 +1249,7 @@ def get_or_create(self, network):
         external_nets = (n for n in self._provider.networking.networks
                          if n.external)
         for net in external_nets:
-            if self._check_fip_connectivity(network, net):
```
Comment (nuwang) [inline cloudbridge/providers/openstack/services.py:1252]: We can remove the `_check_fip_connectivity` related functions entirely?

Comment (almahmoud) [reply cloudbridge/providers/openstack/services.py:1252]: We could, I just thought having the function might not hurt if we ever want to check for connectivity for other purposes?

Comment (nuwang) [reply cloudbridge/providers/openstack/services.py:1252]: Doesn't really do anything of significance I think: https://github.com/CloudVE/cloudbridge/blob/d51933a28677e4f2cc2c077fed036c7851766145/cloudbridge/providers/openstack/services.py#L1229
Let's remove and add back if necessary.

Comment (almahmoud) [reply cloudbridge/providers/openstack/services.py:1252]: Done

**Review by nuwang — APPROVED** [review]

**Pull request merged**